wdio/specs/contacts.js

```diff
@@ -29,22 +29,6 @@ describe('Municipality contacts page', () => {
     // Make sure other contacts disappear when searching.
     assert(!browser.isVisible('h4=נסר בו סריחאן'));
     assert(!browser.isVisible('h4=סלאח אבו האני'));
-
-    // should find contacts searching for department.
-    input.setValue('חינ');
```

Comment:
Removed those because they are causing a lot of problems with WDIO, it doesn't seem to fill the field properly and it's too many scenarios that one of them is going to fail in the 3 tries, we have the same tests in Elm, so I don't they are needed.